bundle.osl/manifests/logic-operator-rhel8-builder-config_v1_configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+data:
+  DEFAULT_BUILDER_RESOURCE_NAME: Dockerfile
+  DEFAULT_WORKFLOW_EXTENSION: .sw.json
+  Dockerfile: "FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-swf-builder-rhel8:latest
```

Comment:
Isn't that dangerous to use `latest` ? I would set it to current prod version, aka `1.30`

Reply:
Since we are on main as discussed we could use this and switch on each version branch to the right version